test/test_testing.py

```diff
@@ -1684,10 +1672,10 @@ def test_op_parametrized(self, device, dtype, op, flag):
         expected_test_names = []
         for op in op_db:
             for dtype in op.default_test_dtypes(device):
-                for flag_part in ('_flag_disabled_', '_flag_enabled_'):
-                    op_name = '{}{}'.format(op.name, '_' + op.variant_test_name if op.variant_test_name else '')
-                    part1 = '{}.test_op_parametrized_{}'.format(device_cls.__name__, op_name)
-                    expected_test_names.append(part1 + '_' + dtype_name(dtype) + flag_part + device)
+                for flag_part in ('flag_disabled', 'flag_enabled'):
+                    expected_name = '{}.test_op_parametrized_{}_{}_{}_{}'.format(
+                        device_cls.__name__, op.formatted_name, device, dtype_name(dtype), flag_part)
+                    expected_test_names.append(expected_name)
```

Comment:
In `test_name_fn`, the device string is at the very end. In `test_ops_composition_names`, the device string is somewhere in the middle (and before the parametrized args). Is this intentional? Would it be better to make it consistent?